### PR TITLE
Fix doc

### LINKED ISSUE #341: Doc: Arithmetic clarifications
In the [Spectral Cube Arithmetic](http://spectral-cube.readthedocs.io/en/latest/arithmetic.html) section, it is cautioned that copies of the whole cube will be made. However, is this still true if I only want to do arithmetic on slices of cubes? For example:

```python
cube2 = cube[0, :, :] * 2
```

Will this make a copy only of `cube[0, :, :]` or the entire `cube`?

### LINKED ISSUE #343: Doc: Subset calc example
In the section [Work with small subsets of data at a time](http://spectral-cube.readthedocs.io/en/latest/big_data.html#work-with-small-subsets-of-data-at-a-time), I think

```python
y = x[0] * 0
```

is more readable as

```python
y = np.zeros_like(x[0])
```

### PR DESCRIPTION
**Pull request by pllim**
Changes include:

* Fix typo and syntax errors preventing proper text rendering.
* Fixes #341
* Fixes #343
* Address the doc portion of #339 and added a note to clarify mask convention.

### CODE REVIEW

**Comment by pllim** [comment]
Overlaps slightly with #342

File: docs/masking.rst [hunk 1]
```diff
@@ -148,9 +157,11 @@ be converted into a CASA image using :func:`~spectral_cube.io.make_casa_mask`:
   >>> from spectral_cube.io.casa_masks import make_casa_mask
   >>> make_casa_mask(cube, 'casa_mask.image', add_stokes=False)  # doctest: +SKIP
 
-Optionally, a redundant Stokes axis can be added to match the original CASA image.
-.. Masks may also be appended to an existing CASA image:
-..  >>> make_casa_mask(cube, 'casa_mask.image', append_to_img=True, img='casa.image')
+Optionally, a redundant Stokes axis can be added to match the original CASA
+image. Masks may also be appended to an existing CASA image::
+
+  >>> make_casa_mask(cube, 'casa_mask.image', append_to_img=True,
```
Comment (keflavich) [inline docs/masking.rst]: These lines were meant to be commented out, but the comments were improperly formatted.  At the moment, I'm not sure if we ever finished implementing the `append_to_img` functionality... we may have, but if not, we should actually comment this bit out.

Comment (pllim) [inline docs/masking.rst]: I commented them out properly.

**Comment by keflavich** [comment]
:+1:, though there are build failures from the doctests.... probably caused by #342

**Comment by astrofrog** [comment]
>  probably caused by

I think you mean **thanks to** ;)

**Comment by pllim** [comment]
It was my bad. I marked the offending line to be skipped by `doctest`, now only 1 `yt` related test is failing.

**Comment by astrofrog** [comment]
Looks good to me!

**Pull request merged**